examples/react-native-vanilla/storybook/stories/index.js

```diff
@@ -13,13 +13,13 @@ storiesOf('Welcome', module).add('to Storybook', () => <Welcome showApp={linkTo(
 
 storiesOf('Button', module)
   .addDecorator(getStory => <CenterView>{getStory()}</CenterView>)
-  .add('with text', () => (
+  .add('with text', () =>
     <Button onPress={action('clicked-text')}>
       <Text>Hello Button</Text>
     </Button>
-  ))
-  .add('with some emoji', () => (
+  )
+  .add('with some emoji', () =>
```

Comment:
I think the idea of prettier is you don't configure it, you just enjoy not quibbling over this stuff.